output/Form.md

```diff
@@ -40,7 +40,40 @@ title: Yerun Open Science Award form
 
 -   *Can the practice be reused in other departments or institutions?*
 
-<!--# blank field, max 400 words -->
+The idea is to record all the steps that are taken so that they are accessible
+and reusable by everyone, regardless of their level of experience with the
+technology. Thus, in our research group we are training each of the members in
+the use of tools that allow us to be more transparent and compliant with open
+science requirements in relation to our scientific production through seminars,
+such as:
+
+*The creation of projects in RStudio with a template that can be modified for
+each study we are involved.* The use of version control provided by GitHub. *The
+use of remote servers (e.g., CCC)* The use of other programming frameworks such
```

Comment:
CCC is the acronym for "Centro de Computación Científica", the assessment panel won't know what it means.